lib/Parser/Parse.cpp

```diff
@@ -7004,6 +7010,12 @@ void Parser::ParseFncFormals(ParseNodeFnc * pnodeFnc, ParseNodeFnc * pnodeParent
 
             if (flags & fFncOneArg)
             {
+                if (seenRestParameter)
```

Comment:
moving this down - will it change the line number/column number of the error when that is reported?

Reply:
or this part of the for loop? 

---
In reply to: [279027586](https://github.com/Microsoft/ChakraCore/pull/6111#discussion_r279027586) [](ancestors = 279027586)

Reply:
Yes, this change results in the wrong column number being reported.

Great catch!